app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/PasscodeSettingsFragment.kt

```diff
@@ -5,15 +5,17 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.navigation.Navigation
+import androidx.navigation.fragment.findNavController
+import io.gnosis.data.models.Safe
 import io.gnosis.safe.ScreenId
 import io.gnosis.safe.databinding.FragmentSettingsAppPasscodeBinding
 import io.gnosis.safe.di.components.ViewComponent
-import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import io.gnosis.safe.ui.base.SafeOverviewBaseFragment
 import io.gnosis.safe.ui.settings.app.SettingsHandler
 import pm.gnosis.svalinn.common.utils.visible
 import javax.inject.Inject
 
-class PasscodeSettingsFragment : BaseViewBindingFragment<FragmentSettingsAppPasscodeBinding>() {
+class PasscodeSettingsFragment : SafeOverviewBaseFragment<FragmentSettingsAppPasscodeBinding>() {
```

Comment:
this doesn't have to be SafeOverviewBaseFragment. BaseViewBindingFragment should be enough

Reply:
The snackbars are in SafeOverviewBaseFragment. Can we move them to BaseViewBindingFragment?

Reply:
I would prefer to keep BaseViewBindingFragment as simple as possible. We can leave it as SafeOverviewBaseFragment for now and move snackbar handling to a separate helper later